src/ServiceFabric/ServiceFabric/ChangeLog.md

```diff
@@ -18,6 +18,9 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Fix add node type cmdlet bugs:
```

Comment:
Can we add any test validation / regression protection for this fix to the ServiceFabric tests?

Reply:
sure, I'll add the test

Reply:
I recorded the tests again adding the necessary things so the fixes I made were validated